priv/gettext/ru/LC_MESSAGES/apns.po

```diff
@@ -83,13 +83,13 @@ msgstr ""
 
 #, elixir-format
 #: lib/t/push_notifications/apns.ex:83
-msgid "Invite your match to a date if you want to keep it alive ✨"
+msgid "Invite your match %{name} to a date if you want to keep it alive ✨"
 msgstr "Пригласи на дэйт мэтча, если хочешь его сохранить ✨"
```

Comment:
why EN has %{name} and RU doesn't?

Reply:
fixed

Reply:
![image](https://user-images.githubusercontent.com/16062322/142439658-8546d14d-ffd4-4d6f-82e0-4ff0ff0dbe2e.png)

actually both of them shouldn't have %{name}

Reply:
agree with you, deleted one of these

Reply:
![image](https://user-images.githubusercontent.com/16062322/142568496-9176e517-ece4-4ad4-90c2-d5dc2a86fef8.png)
it's still there

Reply:
seems like I forget to commit, sorry again